src/components/tree-item/tree-item.scss

```diff
@@ -102,10 +102,10 @@
 // focus styles
 :host {
   @apply focus-base;
-  &:focus,
-  &:active {
-    @apply focus-inset outline-none;
-  }
+}
+
+:host(:focus:not([disabled])) {
```

Comment:
Are focus styles applied when the item is disabled? IIRC, for that case we block pointer events, prevent tabbing and will blur the item if it becomes disabled and already had focus.